src/Microsoft.DotNet.ProjectJsonMigration/ProjectDependencyFinder.cs

```diff
@@ -50,12 +61,12 @@ public IEnumerable<ProjectDependency> ResolveProjectDependencies(ProjectContext
 
         private Dictionary<string, ProjectDependency> FindPossibleProjectDependencies(string projectJsonFilePath)
         {
-            var projectDirectory = Path.GetDirectoryName(projectJsonFilePath);
+            var projectRootDirectory = GetRootFromProjectJson(projectJsonFilePath);
 
             var projectSearchPaths = new List<string>();
-            projectSearchPaths.Add(projectDirectory);
```

Comment:
no we don't need to, looks like nuget doesn't

https://github.com/NuGet/NuGet.Client/blob/125b3bbd41dec25abfae8dd8d64627f78aae41d0/src/NuGet.Core/NuGet.ProjectModel/PackageSpecResolver.cs#L93